tsconfig.json

```diff
@@ -16,11 +16,4 @@
     "skipLibCheck": true,                                /* Skip type checking all .d.ts files. */
     "composite": true,
   },
-  "include": [                                         /* Specifies an array of filenames or patterns to include in the program. */
```

Comment:
These didn't do anything because it was actually using `include` from the `tsconfig.build.json` files.